package.json

```diff
@@ -28,6 +28,7 @@
   },
   "dependencies": {
     "analyze-css": "^2.0.0",
+    "analyze-image": "gmetais/analyze-image#main",
```

Comment:
As you've mentioned in the PR description - let's either publish this package to npm (ideally) or at least pin the github dependency to a specific commit / tag.